boxes/token/src/contracts/src/types/transparent_note.nr

```diff
@@ -1,10 +1,6 @@
 // docs:start:token_types_all
 use dep::aztec::{
-    note::{
-        note_header::NoteHeader,
-        note_interface::NoteInterface,
-        utils::compute_note_hash_for_consumption,
-    },
+    note::utils::compute_note_hash_for_consumption,
     hash::{compute_secret_hash, pedersen_hash},
     context::PrivateContext,
```

Comment:
think this is also added in the prelude?

Reply:
Oops missed that!

Reply:
Removed